CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ that feature *on top of* openHAB.
 
 ### Discuss your design on the mailing list
```

Comment:
Should we also update thuis header?

Reply:
That's a good one. AFAIK there is no mailing list. I haven't read all forum posts, but it looks to me that most design discussions nowadays take place in issues and PRs on GitHub.